README.md

```diff
@@ -60,17 +60,40 @@ import {Button, Modal, launchModal, closeModal} from '@getalby/bitcoin-connect-r
 closeModal();
 ```
 
-#### React SSR / NextJS
-
-Make sure to only render the components **client side**. This can be done either by creating a wrapper component with 'use client' directive (NextJS app directory), using next/dynamic, or a dynamic import e.g.
-
-```ts
-useEffect(() => {
-  // init bitcoin connect to provide webln
-  import('@getalby/bitcoin-connect-react');
-}, []);
+#### NextJS / SSR
+
+Make sure to only import and render the components **client side**. This can be done either by creating a wrapper component with using next/dynamic with `ssr: false` (and add the 'use client' directive when using the NextJS app router), or a dynamic import e.g.
+
+```tsx
+"use client"
```

Comment:
Is this needed? I didn't :smile: 

I just did this in a file:

```js
import dynamic from 'next/dynamic'

const Modal = dynamic(
  () => import('@getalby/bitcoin-connect-react').then((mod) => mod.Modal),
  {
    ssr: false,
  }
)
```

and it worked

Reply:
@ekzyis I looked at stacker news and it's not using the App Router yet, so it's not needed. If you migrate to use the App Router you'll need that line.

Reply:
> If

Big if, lol :)

Reply:
It's a few lines up (line 65) in this PR. But I understand most people won't read everything in the README. But it's also in the App Router demo variant.